blazor/datagrid/filtering.md

```diff
@@ -159,4 +159,91 @@ Use the [GetFilteredRecordsAsync](https://help.syncfusion.com/cr/blazor/Syncfusi
     }
 }
 
+```
+
+## Filter enum column
+
+You can filter the enum type data in the grid column using the Filter Template feature of the Grid.
```

Comment:
Filter the enum type data in the grid column using the Filter Template feature of the Grid.